src/reducers/message.ts

```diff
@@ -247,45 +247,46 @@ function handleCombinePoints(
   state: MessageState,
   action: Action<CombinePointsParams>
 ): MessageState {
+  const withinBounds = (index: number): boolean => {
+    return index >= 0 && index < state.points[action.params.shape].length;
+  };
+
+  const isQuoted = (index: number): boolean => {
+    return state.points[action.params.shape][index].quotedAuthor !== undefined;
```

Comment:
What if quotedAuthor is null (or an empty string, etc.) for some reason? Would it be alright to instead return:
`!!state.points[action.params.shape][index].quotedAuthor`

Reply:
Yep, thanks for the catch